cspatients/views.py

```diff
@@ -73,18 +72,8 @@ def form(request):
     }
     status_code = 200
     if request.method == "POST":
-        status_code = 400
-        # Send the Form information
-        patient = PatientSerializer(data=request.POST)
-        patiententry = PatientEntrySerializer(data=request.POST)
-        if patient.is_valid():
-            patient.save()
-            if patiententry.is_valid():
-                patiententry.save()
-                context = {
-                    "saved": True,
-                }
-                status_code = 201
+        status_code = save_model(request.POST)
+        if status_code == 201:
             send_consumers_table()
```

Comment:
Your view should really be the thing that handles your response to a request. I don't think delegating that to a util method is good practise.
If multiple views make use of this method then rather put it in the views.py file (I know this is contrary to what I told you when we discussed it but I think its the better option)
Or is this a channels thing?  since you seem to `print` the data in the method?